src/mumble/MainWindow.cpp

```diff
@@ -1706,13 +1710,15 @@ void MainWindow::on_qaUserLocalIgnoreTTS_triggered() {
 	p->setLocalIgnoreTTS(ignoredTTS);
 	if (! p->qsHash.isEmpty())
 		g.db->setLocalIgnoredTTS(p->qsHash, ignoredTTS);
+	else
+		msgBox(MainWindow::tr("This change could not be saved permanently and is lost on restart because %1 does not have a certificate.").arg(Log::formatClientUser(p, Log::Target)));
 }
 
 void MainWindow::on_qaUserLocalVolume_triggered() {
 	ClientUser *p = getContextMenuUser();
-	if (!p) {
```

Comment:
This has obviously nothing to do with the PR but I saw that it was inconsistent to the other slots and changed.